streams/src/test/java/org/apache/kafka/streams/kstream/SuppressedTest.java

```diff
@@ -46,13 +48,13 @@ public void bufferBuilderShouldBeConsistent() {
         assertThat(
             "keys alone should be set",
             maxRecords(2L),
-            is(new EagerBufferConfigImpl(2L, MAX_VALUE))
+            is(new EagerBufferConfigImpl(2L, MAX_VALUE, Collections.emptyMap()))
```

Comment:
@vichu Can you add some tests that don't just use an empty map?

Reply:
Sure. 
Added a few tests to verify the `logConfig` map as well.